mmv1/products/cgc/terraform.yaml

```
@@ -230,6 +230,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_private_ip"
+        primary_resource_id: "instance"
+        primary_resource_type: "google_sql_database_instance"
+        vars:
+          private_network: "private-network"
+          private_ip_address: "private-ip-address"
+          private_ip_sql_instance: "private-ip-sql-instance"
+        ignore_read_extra:
+          - "root_password"
```

```suggestion
          - "root_password"
          - "deletion_protection"
```
also need the deletion_protection field here